src/tests/multiPointCreate.test.tsx

```diff
@@ -30,7 +30,7 @@ describe("remove shape in non linear elements", () => {
     fireEvent.pointerDown(canvas, { clientX: 30, clientY: 20 });
     fireEvent.pointerUp(canvas, { clientX: 30, clientY: 30 });
 
-    expect(renderScene).toHaveBeenCalledTimes(3);
+    expect(renderScene).toHaveBeenCalledTimes(4);
```

Comment:
Sadpanda. Do we need to render the scene one more time?

Reply:
We'd need to pass the offsets from upstream.

Reply:
Although I suppose we can update state only if the values are different (`setState()` doesn't bail on its own).